DashDocs/src/chapters/dash_vtk/IntroToVisualization/examples/index.jl

```diff
@@ -0,0 +1,109 @@
+@doc_chapter "/dash_vtk/intro_to_visualization" begin 
+
+@layout html_div() do 
+
+    html_h1("Intro to 3D Visualization (3D Visualization Explained)"),
+
+    dcc_markdown("
+    In VTK, we have 3 main types of objects that are key for understanding its visualization principals. First we have the *View* which is just a container for any *Representation* of DataSource that you want to see.
+
+    ## View
+    The view is a 3D View that can do Geometry rendering for meshes or Volume rendering for 3D images. The view can be configured to act as a 2D one when using parallel projection and preventing rotation when interacting with it. The *View* component can be configured with the following set of properties.
+
+    ```
+    dash_vtk.View(
+      id='vtk-view',

```

Comment:
This is not Julia syntax. Please make sure this is runnable Julia code.

Reply:
Yes, I am looking into it and will keep in mind further. Thanks for the review.